examples/07-EMIT/interference_classification.py

```diff
@@ -0,0 +1,107 @@
+from pyaedt.emit_core.emit_constants import InterfererType, ResultType, TxRxMode
+from pyaedt import Emit
+import pyaedt
+import sys
+
+def interference_classification(tx_radios, rx_radios, radios, emitapp, rev, domain, use_filter, filter):
+    """
+    Classisify interference type as according to inband/inband, 
+    out of band/in band, inband/out of band, and out of band/out of band.
+
+    Parameters
+        ----------
+        tx_radios : string list of tx radio names
+        rx_radios : string list of rx radio names
+        radios : list of radio objects
+        emitapp : instance of Emit
+        rev : emitapp results
+        domain : interaction domain of emit results
+        use_filter : boolean value, True if filtering is being used
+        filter : list of filter values if filtering is in use
+
+        Returns
+        -------
+        all_colors : list of colors for classification of interterference types
+        power_matrix : list of worst case interference power at Rx
+    """
+    power_matrix = []
+    all_colors = []
+    modeRx = TxRxMode.RX
+    modeTx = TxRxMode.TX
+    mode_power = ResultType.POWER_AT_RX
+
+    for tx_radio in tx_radios:
+        rx_powers = []
+        rx_colors = []
+        for rx_radio in rx_radios:
+            # powerAtRx is the same for all Rx bands, so just use first one
+            rx_bands = rev.get_band_names(rx_radio, modeRx)
+            rx_band_objects = radios[rx_radio].bands()
+            if tx_radio == rx_radio:
+                # skip self-interaction
+                rx_powers.append('N/A')
+                rx_colors.append('white')
+                continue       
+
+            max_power = -200
+            tx_bands = rev.get_band_names(tx_radio, modeTx)
+            tx_band_objects = radios[tx_radio].bands()
+
+            for i in range(len(rx_bands)):
+                # Find the highest power level at the Rx input due to each Tx Radio.
+                # Can look at any Rx freq since susceptibility won't impact
+                # powerAtRx, but need to look at all tx channels since coupling
+                # can change over a transmitter's bandwidth
+                rx_freq = rev.get_active_frequencies(rx_radio, rx_bands[i], modeRx)[0]
+                rx_start_freq = radios[rx_radio].band_start_frequency(rx_band_objects[i])
+                rx_stop_freq = radios[rx_radio].band_stop_frequency(rx_band_objects[i])
+                rx_channel_bandwidth = radios[rx_radio].band_channel_bandwidth(rx_band_objects[i])
```

Comment:
Since we stumbled on this initially, I think we should add a comment explaining the difference between `rx_freq` and start/stop freqs. Something like (feel free to change):

> The start and stop frequencies define the Band's extents, while the active frequencies are a subset of the Band's frequencies being used for this specific project as defined in the Radio's Sampling.

Reply:
Sounds good, added that comment